### DISCUSSION TITLE
What is the length of the rubber band? 📏

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1212] [Galaxy Trivia] [answered]
Three circles of equal size are placed in a triangle, and they are pairwise tangent. A rubber band is placed around the three circles. The height from the top of the top circle to the bottom of the bottom two circles is exactly 2 meters.

<img width="686" alt="image" src="https://github.com/ParanoidUser/yolo/assets/5120290/f79e3479-36a2-4ff8-b887-607bf9e788eb">

**Comment by peterbaccaro** [comment, marked as answer]
The length of the rubber band is **6.8 meters**.

**Reply by ParanoidUser** [reply]
Sounds about right. Just a little over the top, but could be a rounding error. 🧮
